### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by jonasNobile**

### CODE REVIEW

**Review by honzasladek — APPROVED** [review]
👍

File: headless-cms/README.md [hunk 1]
```diff
@@ -57,3 +57,35 @@ Then select:
...
+<ApplicationEntrypoint
+	...
+	defaultLocale="cs-CZ"
+	dictionaries={{
+		"cs-CZ": csCZ,
+	}}
+/>
+```
+
+#### Setting the language for custom components
```
Comment (VojtaStanek) [inline headless-cms/README.md:81]:
```suggestion
#### Setting the language for custom components and labels
```

File: headless-cms/README.md [hunk 2]
```diff
@@ -57,3 +57,35 @@ Then select:
 - Are you sure you want to create this API key?: `y` (yes)
 
 Set `NEXT_PUBLIC_TOKEN` in `head/.env` to newly generated API token (replace the default superadmin token) and restart Docker containers (`docker-compose restart`).
+
+### 🌍 Localization
+
+The default language of administration is English. Other supported language is Czech (more languages coming soon). To change the language you just need to do a few steps.
```
Comment (VojtaStanek) [inline headless-cms/README.md:63]:
```suggestion
The default language of administration is English. Other supported language is Czech, but you can add any other language. To change the language you just need to do a few steps.
```

File: headless-cms/admin/components/ContentField.tsx [hunk 3]
```diff
@@ -21,28 +25,53 @@ const inlineButtons: BlockEditorProps["inlineButtons"] = [
 		RB.headingOne,
 		RB.headingTwo,
 		RB.headingThree,
-		RB.headingFour,
+		RB.headingFour
```
Comment (VojtaStanek) [inline headless-cms/admin/components/ContentField.tsx:28]: 🤡

File: headless-cms/admin/locales/csCZ.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+	"OG title": "OG titulek",
+	"Page role": "Role stránky",
+	"Pages": "Stránky",
+	"Perex": "Perex",
+	"Phone number": "Telefonní číslo",
+	"Position": "Pozice",
+	"Preview": "Náhled",
+	"Primary": "Primární",
+	"Publish date": "Datum zveřejnění",
+	"Recommended aspect ratio 19:10 (e.g.: 2400×1260 px).": "Doporučený poměr 19:10 (např. 2400×1260 px).",
```
Comment (VojtaStanek) [inline headless-cms/admin/locales/csCZ.ts:65]:
```suggestion
	"Recommended aspect ratio 19:10 (e.g.: 2400×1260 px).": "Doporučený poměr stran 19:10 (např. 2400×1260 px).",
```

File: headless-cms/admin/components/ContentField.tsx (hunk 3 above)
Comment (jonasNobile) [reply headless-cms/admin/components/ContentField.tsx:28]: It's correct in second PR. It'll be right after rebase.

File: headless-cms/admin/locales/csCZ.ts [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+	"Select what you want to link.": "Vyberte kam chcete odkazovat.",
+	"SEO articles": "SEO články",
+	"SEO pages": "SEO stránky",
+	"SEO": "SEO",
+	"Settings": "Nastavení",
+	"Short summary of the page. Keep it between 120 and 158 characters.": "Krátký popis stráky. Ideální rozsah je mezi 120 a 158 znaky.",
+	"Show contact message": "Zobrazit zprávu",
+	"Subtitle": "Podtitul",
+	"Testimonial section": "Sekce se zkušentmi",
+	"Testimonials": "Zkušenosti",
```
Comment (VojtaStanek) [inline headless-cms/admin/locales/csCZ.ts:78]: 🤨 

https://cs.wikipedia.org/wiki/Testimoni%C3%A1l

File: headless-cms/admin/locales/csCZ.ts [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+	"Recommended aspect ratio 19:10 (e.g.: 2400×1260 px).": "Doporučený poměr 19:10 (např. 2400×1260 px).",
+	"Remove link": "Odstranit odkaz",
+	"Role": "Role",
+	"Secondary": "Sekundární",
+	"Select what you want to link.": "Vyberte kam chcete odkazovat.",
+	"SEO articles": "SEO články",
+	"SEO pages": "SEO stránky",
+	"SEO": "SEO",
+	"Settings": "Nastavení",
+	"Short summary of the page. Keep it between 120 and 158 characters.": "Krátký popis stráky. Ideální rozsah je mezi 120 a 158 znaky.",
```
Comment (VojtaStanek) [inline headless-cms/admin/locales/csCZ.ts:74]:
```suggestion
	"Short summary of the page. Keep it between 120 and 158 characters.": "Krátký popis stránky. Ideální rozsah je mezi 120 a 158 znaky.",
```

File: headless-cms/admin/locales/csCZ.ts (hunk 5 above)
Comment (jonasNobile) [reply headless-cms/admin/locales/csCZ.ts:78]: `Doporučení` is probably the best one.

**Pull request merged**